ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,59 @@
+<!--
+Compensation requests have a specific format that enables them to be parsed programmatically for more efficient budgeting. Please make sure you stick to the template. A linter will evaluate your request after you remove [WIP] from the issue title and notify you of any issues.
```

Comment:
'linter' is used here, 'parser' is used below. Might be best to use a single term, since it is a single bot doing the work.

Reply:
True but there are 2 processes -- linting before voting and then parsing after voting, which is why 2 terms were used. But I agree a consistent term would probably be clearer for users so I've changed it.